framework/src/play/templates/TemplateLoader.java

```diff
@@ -217,6 +221,46 @@ public static Template load(String path) {
                 }
             }
         }
+
+        String play_compile_templates = Play.configuration.getProperty(
+                "play_compile_templates",
+                System.getProperty(
+                        "play_compile_templates",
+                        System.getenv("PLAY_COMPILE_TEMPLATES")));
+        String play_compile_templates_path_separator = Play.configuration.getProperty(
+                "play_compile_templates_path_separator",
```

Comment:
We used '.' to separate word in config param name so should be @play.compile.templates.path.separator@